tests/oeedger8r/behavior/switchless_trusted.edl

```diff
@@ -4,6 +4,6 @@
 enclave {
     trusted {
         // Switchless functions not supported.
```

Comment:
But now maybe supported?!

Reply:
Not yet. We are supporting switchless ocalls first.